lib/uwb/protocols/fira/UwbOobConversions.cxx

```diff
@@ -142,4 +144,192 @@ uwb::protocol::fira::GetUciConfigParams(const UwbConfiguration& uwbConfiguration
+            builder.SetSlotDuration(std::get<uint16_t>(parameterValue));
+            break;
+        case UwbApplicationConfigurationParameterType::RangingInterval:
+            builder.SetRangingInterval(std::get<uint32_t>(parameterValue));
+            break;
+        case UwbApplicationConfigurationParameterType::StsIndex:
+            // TODO
+            break;
+        case UwbApplicationConfigurationParameterType::MacFcsType:
+            builder.SetMacAddressFcsType(std::get<UwbMacAddressFcsType>(parameterValue));
+            break;
+        case UwbApplicationConfigurationParameterType::RangingRoundControl:
+            // TODO
+            break;
+        case UwbApplicationConfigurationParameterType::AoAResultRequest:
+            // TODO
+            break;
+        case UwbApplicationConfigurationParameterType::RangeDataNotificationConfig:
+            // TODO
+            break;
+        case UwbApplicationConfigurationParameterType::RangeDataNotificationProximityNear:
+            // TODO
+            break;
+        case UwbApplicationConfigurationParameterType::RangeDataNotificationProximityFar:
+            // TODO
+            break;
+        case UwbApplicationConfigurationParameterType::DeviceRole:
+            builder.SetDeviceRole(std::get<DeviceRole>(parameterValue));
+            break;
+        case UwbApplicationConfigurationParameterType::RFrameConfiguration:
+            builder.SetStsPacketConfiguration(std::get<RFrameConfiguration>(parameterValue));
+            break;
+        case UwbApplicationConfigurationParameterType::PreambleCodeIndex:
+            builder.SetPreambleCodeIndex(std::get<uint8_t>(parameterValue));
+            break;
+        case UwbApplicationConfigurationParameterType::SfdId:
+            // TODO
+            break;
+        case UwbApplicationConfigurationParameterType::PsduDataRate:
+            // TODO
+            break;
+        case UwbApplicationConfigurationParameterType::PreambleDuration:
+            // TODO
+            break;
+        case UwbApplicationConfigurationParameterType::RangingTimeStruct:
+            builder.SetRangingTimeStruct(std::get<RangingMode>(parameterValue));
+            break;
+        case UwbApplicationConfigurationParameterType::SlotsPerRangingRound:
+            builder.SetMaxSlotsPerRangingRound(std::get<uint8_t>(parameterValue));
+            break;
+        case UwbApplicationConfigurationParameterType::TxAdaptivePayloadPower:
+            // TODO
+            break;
+        case UwbApplicationConfigurationParameterType::ResponderSlotIndex:
+            // TODO
+            break;
+        case UwbApplicationConfigurationParameterType::PrfMode:
+            if (std::get<PrfModeDetailed>(parameterValue) == PrfModeDetailed::Bprf62MHz) {
+                builder.SetPrfMode(PrfMode::Bprf);
+            } else { // PrfModeDetailed::Hprf124MHz || PrfModeDetailed::Hprf249MHz
+                builder.SetPrfMode(PrfMode::Hprf);
+            }
```

Comment:
Would rather see the explicit check here so future changes (however unlikely) don't cause this to stop working if a newly added enum value is used.
```suggestion
            auto prfModeValue = std::get<PrfModeDetailed>(parameterValue);
            switch (prfModeValue) {
            case PrfModeDetailed::Bprf62MHz:
                builder.SetPrfMode(PrfMode::Bprf);
                break;
            case PrfModeDetailed::Hprf124MHz: [[ fallthrough ]];
            case PrfModeDetailed::Hprf249MHz:
                builder.SetPrfMode(PrfMode::Hprf);
                break;
            default:
                PLOG_WARNING << "unsupported PRF Mode value " << magic_enum::enum_name(prfModeValue) << " specified; ignoring";
                break;
            }
```